### DISCUSSION TITLE
"Maximum extrusion length" deprecated ? replaced by "Maximum G1 per second" ? no !

### DISCUSSION THREAD

**Discussion by DzzD** [supermerill/SuperSlicer#1863] [Q&A] [answered]
Hello,

I saw in the last version of super slicer that "Maximum extrusion length" is being deprecated and replaced by "Maximum G1 per second", I think that both are useful and should stay in place.

Imagine a slicing producing a lot of G1 movements lower than 0.0001mm (maybe a hyper detailled model). Those G1 will may be not being removed by "Maximum G1 per sec". If the whole G1 rate stay below this parameter. Even if those movments have no sens.

Also Maximum G1 should be Maximum G1/G2/G3 because I am sure you will integrate ArcWelder  soon, no :) ?

So, finally and IMHO, I reallly think the "Maximum extrusion length" parameter should stay in place.

**Comment by supermerill** [comment, marked as answer]
The hyper detailed model should be 'cut' by the resolution option.
But ok, i'll keep it.

**Comment by rraallvv** [comment]
I couldn't find any settings related to the Maximum Extrusion Length (SS v 2.5.5913). When printing multiple consecutive long lines, and the filament runout sensor triggers a pause, it seems the printer only pauses after completing all the G-code comands in the cache (or some other kind of FIFO memory). By then, the remaining filament that usually sticks out of the extruder has already been pulled in, making it difficult to replace with a new spool. The solution would be to split G-code with long lines into multiple shorter segments.
